_data/ols-6-projects.yaml

```diff
@@ -135,23 +135,7 @@
   name: Open Data Custodianship tool
   participants:
   - dingaaling
-- description: Generation Z stands to become the new entrants into the world of research
```

Comment:
The description for this project is before, on line 117 :smile:

Reply:
oh my :/... I'll check